packs/st2/actions/actions_list.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Once we move st2client in a separate repo and refactor the code so it doesn't abuse kwargs so we can introspect the code, we can auto generate most of those actions.